run.go

```diff
@@ -58,6 +58,8 @@ type RunOptions struct {
 	User string
 	// WorkingDir is an override for the working directory.
 	WorkingDir string
+	// Shell is default shell to run in a container
```

Comment:
Missing `.` at the end of this sentence.

Reply:
Fixed